test/helpers/mock_shell.rb

```diff
@@ -0,0 +1,17 @@
+module MockShell
+  class << self
+    # For easy testing, this method just returns back the command it is given.
+    # The real Shell class will return the output string.
+    def execute(command)
+      return command
+    end
+
+    def ifconfig(iface = '')
+      self.execute("LANG=en && inconfig #{iface}")
```

Comment:
Yep, that's a typo, nice catch. :) That's just with the mock shell at least, that line will only ever get logged rather than executed. I'll push a fix!